src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -122,7 +122,45 @@ <h1 id="enroll-header" class="text-break">Enroll Students for {{ courseId }}</h1
     </tm-loading-retry>
     <button id="paste" [hidden]="true" (click)="showPasteModalBox()"></button> <!-- Acts as a medium to send the modal box into the JS file -->
 
-    <div class="more-info" #moreInfo [hidden]="showEnrollResults">
+    <div id="results-panel" *ngIf="coursePresent && showEnrollResults">
+      <h1 class="text-break">Enrollment Results for {{ courseId }}</h1>
+      <div *ngFor="let enrollResultPanel of enrollResultPanelList">
+        <div class="card enroll-results-panel" *ngIf="enrollResultPanel.studentList.length > 0">
+          <div class="card-header" [ngClass]="{'bg-success': enrollResultPanel.status === EnrollStatus.NEW,
+       'bg-warning': enrollResultPanel.status === EnrollStatus.MODIFIED,
+       'bg-secondary': enrollResultPanel.status === EnrollStatus.MODIFIED_UNCHANGED,
+       'bg-danger': enrollResultPanel.status === EnrollStatus.ERROR,
+       'bg-default': enrollResultPanel.status === EnrollStatus.UNMODIFIED}" [style.color]="enrollResultPanel.status ===
+       EnrollStatus.NEW || enrollResultPanel.status === EnrollStatus.MODIFIED_UNCHANGED ? 'white' : 'black' ">
```

Comment:
nit: indentation of these lines seems a little off, can refer to how they were indented originally. Rest of the PR looks good.

Reply:
`tr` elements need an extra level of indent as well.

Reply:
Fixed in https://github.com/TEAMMATES/teammates/pull/11872/commits/c260361a2f9298f58367d5b830f8744542255646